pkgs/development/python-modules/jsonschema-spec/default.nix

```diff
@@ -35,11 +46,12 @@ buildPythonPackage rec {
     pathable
     pyyaml
     referencing
-    typing-extensions
+    requests
   ];
 
   nativeCheckInputs = [
     pytestCheckHook
+    responses
```

Comment:
I thought we don't add dependencies twice usually

Reply:
requests and responses are two different packages entirely? :clown_face:

Reply:
oh lol